chain/network/src/peer.rs

```diff
@@ -139,6 +141,9 @@ pub struct Peer {
     pub peer_type: PeerType,
     /// Peer status.
     pub peer_status: PeerStatus,
+    /// Protocol version to communicate with this peer.
+    /// None denotes latest protocol version.
```

Comment:
how is "latest protocol version" defined? why don't we just keep the number?

Reply:
removed the option